### DISCUSSION TITLE
Could not import html table into pad

### DISCUSSION THREAD

**Discussion by merajhossain** [ether/etherpad-lite#4558] [Q&A] [answered]
I use etherpad for collaboration when we do meeting. It is a amazing collaborate editor and so powerful. There have a problem when we insert a html table by setHTML() Api. Editor remove our table tag and generate this type of data see below. For this reason i lost my table data and table style. I want see html table on pad with data. Plz  help me and suggest some way to solve this problem.
![117560947B7C4587AF73F1C8F2B01EEA](https://user-images.githubusercontent.com/19662023/101688640-81ac7300-3a96-11eb-91f0-f290562b1192.png)

**Comment by JohnMcLear** [comment, marked as answer]
Hey.  There is no table support in core.  I suggest you look at table plugins :)

**Reply by merajhossain** [reply]
How i insert a table by etherpad setHTML() api

**Reply by JohnMcLear** [reply]
Look at the plugin, it would be up to the plugin to manage this.

**Reply by ritesshguptaa** [reply]
I am facing the same issue, I tired many Plugins, could  you suggest any plugin that works for it ?
